test/unparse/imp/imp.cfg.kore.out

```diff
@@ -0,0 +1,19 @@
+initGeneratedTopCell[0m ([0m $IO |->[0m on
```

Comment:
What is the `[0m` stuff? terminal escape sequences?

Reply:
these are ansi terminal color codes. I include them in the output because it means we can test the colorization code in our CI testing.